kythe/go/extractors/bazel/settings.go

```diff
@@ -74,6 +74,8 @@ func (s *Settings) SetFlags(f *flag.FlagSet, prefix string) func() {
 		"Only match source paths within the target package")
 	f.BoolVar(&s.Verbose, p("verbose"), false,
 		"Enable verbose (per-file) logging")
+	f.StringVar(&s.ProtoFormat, p("proto_format"), "json",
```

Comment:
There's https://github.com/kythe/kythe/blob/master/kythe/go/platform/tools/kzip/flags/encoding.go ... maybe it needs a more central location first.